File: sample-java/src/main/java/panacea/adapter/activity/MainActivity.java
package panacea.adapter.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import panacea.adapter.R;

/**
 * @author by Arthur Korchagin on 26.04.18.
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.ac_main);
    }
}

File: sample-kotlin/src/main/java/io/github/artkorchagin/kotlin/MainActivity.kt
package io.github.artkorchagin.kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.RecyclerView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.ac_main)
        var mRX: RecyclerView? = null
    }
}
